tools/RAiDER/s1_orbits.py

```diff
@@ -78,6 +79,32 @@ def get_orbits_from_slc_ids(slc_ids: List[str], directory=Path.cwd()) -> List[Pa
     return orb_files
 
 
+def get_orbits_from_slc_ids_hyp3lib(
+    slc_ids: list, orbit_directory: str = None
+) -> dict:
+    """Reference: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/dev/isce2_topsapp/localize_orbits.py#L23"""
+
+    # Populates env variables to netrc as required for sentineleof
+    _ = ensure_orbit_credentials()
+    esa_username, _, esa_password = netrc.netrc().authenticators(ESA_CDSE_HOST)
+    esa_credentials = esa_username, esa_password
+
+    orbit_directory = orbit_directory or 'orbits'
+    orbit_dir = Path(orbit_directory)
+    orbit_dir.mkdir(exist_ok=True)
+
+    orbit_fetcher =  get_orb.downloadSentinelOrbitFile
+
+    orbits = []
+    for scene in slc_ids:
+        orbit_file, _ = orbit_fetcher(scene, str(orbit_dir), esa_credentials=esa_credentials, providers=('ASF', 'ESA'))
```

Comment:
```suggestion
        orbit_file, _ = orbit_fetcher(scene, str(orbit_dir), esa_credentials=esa_credentials, providers=( 'ESA', 'ASF'))
```
ASF's orbit enpoint, s1qc, cannot handle the ARIA processing load, even in the constrained setup we have right now. Since hyp3lib releases tokens when it's done, I'd suggest hitting ESA first

Reply:
Thinking a little more - I am reverting to ASF first again.

https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/dev/isce2_topsapp/localize_orbits.py#L36

I think that's what we do for the topsapp step and that is doing ok. I feel like it shouldn't matter about order, but rather keep it as-is.